LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/mongolian/css3-counter-styles-138.html

```diff
@@ -7,6 +7,7 @@
 <link rel='help' href='http://www.w3.org/TR/css-counter-styles-3/#simple-numeric'>
 <link rel='match' href='css3-counter-styles-138-ref.html'>
 <meta name="assert" content="list-style-type: mongolian produces numbers after 9 per the spec.">
+<meta name=fuzzy content="maxDifference=0-255;totalPixels=10-240">
```

Comment:
The ranges should always start from 0, otherwise this means the test _must_ have at least 10px difference (which I don't think was your intent).

Reply:
I thought to give it a 10 range such that we would be aware if tolerance is no longer needed. It is probably not a good practice, right?

Reply:
The problem is that WPT are shared with other browsers, so this will make tests fail in other browsers with no fuzziness.

Reply:
Fixed.